lib/libc/mips/string/memcpy.c

```diff
@@ -28,25 +28,5 @@
  * SUCH DAMAGE.
  */
 
-#include <cheri/cheri.h>
```

Comment:
Since the contents were replaced, this should be Copyright (c) 2017 Robert Kovacsics